dlc_refine.py

```diff
@@ -0,0 +1,56 @@
+# Extract outliers, refine labels, and merge datasets.
+
+import os
+import subprocess
+import deeplabcut
+import utils
+
+from argparse import ArgumentParser
+
+def set_args():
+    parser = ArgumentParser()
+    parser.add_argument('input_dir', 
+                        type=str,
+                        help="Directory of input videos")
+    parser.add_argument('--video_type',
+                        type=str,
+                        default='mp4',
+                        help="Video extension")
+    parser.add_argument('--outlieralgorithm',
+                        type=str,
+                        default='uncertain',
+                        help="The algorithm used to identify outliers")
+    parser.add_argument('--extractionalgorithm',
+                        type=str,
+                        default='kmeans',
+                        help="The algorithm used to extract frames")
+    parser.add_argument('--gputouse',
+                        type=int,
+                        default=0,
+                        help="GPU name")
+    args = parser.parse_args()
+    return args
+
+
+if __name__ == '__main__':
+    
+    config = utils.load_config()    
+    args = set_args()
+
+    # get input videos
+    videos = [os.path.abspath(os.path.join(args.input_dir, vn)) for vn in os.listdir(args.input_dir)
```

Comment:
I'd make some changes so that it only takes the videos that are readable by the function.